third_party/nvfuser/csrc/ops/alias.h

```diff
@@ -66,4 +66,23 @@ TORCH_CUDA_CU_API TensorView* transpose(
 //! Transpose a 2D tensor.
 TORCH_CUDA_CU_API TensorView* transpose(TensorView* x);
 
+//! Pad a tensor by given widths of zero. Similar to torch.pad, the
+//! pad_widths vector specifies the padding widths of the innermost N
+//! dimensions, where N is half the size of the width vector. Padding
+//! is always done just by zero. TODO: Support other padding types
```

Comment:
Not asking to add support for non-zero padding value in this PR, but supporting non-zero padding value is not hard:
```python
def pad(x, pad_widths, pad_value):
    mask = torch.tensor(True).expand_as(x)
    x_pad_zero = torch.pad(x, pad_widths)
    mask_pad_zero = torch.pad(mask, pad_widths)
    return torch.where(mask_pad_zero, x_pad_zero, pad_value)
```

Reply:
Yeah, the constant padding should be a trivial extension. The other modes of padding, 'reflect', 'replicate' or 'circular', may not be.

Reply:
I think const case makes sense to add for now. Agree the others could be more challenging/interesting to support. I don't think they're required at the moment.

Reply:
Yes, I'll add support of non-zero padding, but I prefer it as a separate PR as this is already rather large.